### PR TITLE
URL encoding

### PR DESCRIPTION
**Pull request by atesgoral**

### CODE REVIEW

**Comment by titaniumbones** [comment]
@dcwalk or @sonalranjit , this is sort of painfully important, if you have a sec to confirm that it works, I'd like to merge ASAP.

File: popup.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function nominationTool( e ) {
   var organizationID = $( '#organizationID' ).val();
   var suborgID = $( '#suborgID' ).val();
   var subprimerID = $( '#subprimerID' ).val();
-  var crawlableID = $( '#crawlableID').val();
+  var crawlableID = $( '#crawlableID:checked' ).val();
```
Comment (dcwalk) [inline popup.js]: unclear why this is included?

Comment (dcwalk) [inline popup.js]: nvm!

**Comment by sonalranjit** [comment]
merging this, this seems to have fixed the docs.google URL not being submitted as well Thanks @atesgoral

**Pull request merged**